scripts/rollup.config.js

```diff
@@ -20,14 +20,15 @@ const options = {
         'm3u8-parser',
         'mpd-parser',
         'mux.js',
-        '@videojs/vhs-utils'
+        '@videojs/vhs-utils',
+        'rollup-plugin-worker-factory'
       ])
     });
   },
   plugins(defaults) {
-    defaults.module.splice(2, 0, 'worker');
-    defaults.browser.splice(2, 0, 'worker');
-    defaults.test.splice(3, 0, 'worker');
+    defaults.module.splice(0, 0, 'worker');
+    defaults.browser.splice(0, 0, 'worker');
+    defaults.test.splice(1, 0, 'worker');
```

Comment:
Just curious if there's a reason this has to be spliced in at 1 (maybe a comment might help)